tutorons/common/api.py

```diff
@@ -0,0 +1,60 @@
+from tastypie.authorization import Authorization
+from tastypie.resources import ModelResource, ALL, ALL_WITH_RELATIONS
+
+from tutorons.common.models import Block, ServerQuery, ClientQuery, Region, ViewedRegion
+from tastypie import fields
+
+
+class BlockResource(ModelResource):
+    class Meta:
+        queryset = Block.objects.all()
+        resource_name = 'block'
+        authorization = Authorization()
+
+
+class ServerQueryResource(ModelResource):
+    class Meta:
+        queryset = ServerQuery.objects.all()
+        resource_name = 'server_query'
+        authorization = Authorization()
```

Comment:
Let's refine authorization.  Ideally, we will allow: only updates to the field for the load finishing time on the client query model; only writes (no reads or updates) to the view events data.

Some references for this include [this Stack Overflow post](http://stackoverflow.com/questions/13704344/tastypie-where-to-restrict-fields-that-may-be-updated-by-patch) and the [TastyPie docs on authorization](https://django-tastypie.readthedocs.org/en/latest/authorization.html)